storage/id_alloc_test.go

```diff
@@ -118,6 +119,7 @@ func TestNewIDAllocatorInvalidArgs(t *testing.T) {
 // 4) Make IDAllocator valid again, the blocked allocations return correct ID.
 // 5) Check if the following allocations return correctly.
 func TestAllocateErrorAndRecovery(t *testing.T) {
+	defer leaktest.AfterTest(t)
 	store, _ := createTestStore(t)
 	allocd := make(chan int, 10)
```

Comment:
Two more lines in my code:
1. defer store.Stop() after createTestStore
2. time.Sleep(50 * time.Millisecond) at the last of TestAllocateErrorAndRecovery

@tschottdorf , could you check whether it is ok?